### DISCUSSION TITLE
Regarding the issue when using pythonnet to call both .NET Framework and .NET Core libraries.

### DISCUSSION THREAD

**Discussion by narasa1234** [pythonnet/pythonnet#2159] [Q&A]
I have multiple libraries created with both .NET Framework and .NET Core. After importing the .NET Framework DLL, I try to import the .NET Core DLL, but I encounter an error during the import. How can I resolve this error?

Also, if I want to import the .NET Framework DLL after importing the .NET Core DLL, what should I do?

### First Import File
```
import clr
clr.Addreference("./NetFrameworkDLL")
```
### Second Import File
```
from pythonnet import load
load("coreclr")　←NG：Failed to initialize pythonnet: System.InvalidOperationException: This property must be set before runtime is initialized
imoprt clr
clr.Addreference("./NetCoreDLL")
```

**Comment by filmor** [comment]
This is not possible. You have to decide.
